lib/ansible/modules/cloud/amazon/ecs_ecr.py

```diff
@@ -292,19 +292,15 @@ def run(ecr, params, verbosity):
 
             elif policy_text is not None:
                 try:
-                    policy = sort_json_policy_dict(policy)
```

Comment:
Minor thing, not a blocker: Since this line is gone, someone with verbosity >= 2 won't get sorted output anymore. I doubt that matters in any significant way, but just something I noticed.